lib/rest.js

```diff
@@ -7,6 +7,18 @@ const assert = require('assert')
 
 class BinanceRest {
 
+    const createQueryObject = (query,addTimeStamp,skipConstruct)=>{
+      if (_.isString(query) && skipConstruct !== true) {

```

Comment:
I think assertObject might be more clear than skipConstruct.  You could put in a little error handling prior to this string check

```javascript
if (assertObject) {
    throw new Error('query must be an object');
}
```